iptables/iptables.go

```diff
@@ -45,6 +45,7 @@ var (
 	iptablesPath  string
 	supportsXlock = false
 	supportsCOpt  = false
+	xLockWaitMsg  = "Another app is currently holding the xtables lock; waiting"
```

Comment:
Yes because I found at least another instance where the first part of the message is printed, but the code is not actually waiting. In that case docker iptables code need to treat it as an error.
See https://git.netfilter.org/iptables/tree/iptables/iptables.c#n1790